src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -221,17 +222,20 @@ void update() {
         Collections.list(treeTable.getColumnModel().getColumns()).forEach(c -> treeTable.removeColumn(c));
 
         // Add back what's appropriate.
-        if (isShowing("Time")) {
+        if (isShowing(TIME)) {
           treeTable.addColumn(tableColumns.get(0));
         }
-        if (isShowing("Sequence")) {
+        if (isShowing(SEQUENCE)) {
           treeTable.addColumn(tableColumns.get(1));
         }
-        if (isShowing("Level")) {
+        if (isShowing(LEVEL)) {
           treeTable.addColumn(tableColumns.get(2));
         }
+        if (isShowing(CATEGORY)) {
+          treeTable.addColumn(tableColumns.get(3));
+        }
 
-        tableColumns.subList(3, tableColumns.size()).forEach(c -> treeTable.addColumn(c));
+        tableColumns.subList(4, tableColumns.size()).forEach(c -> treeTable.addColumn(c));
```

Comment:
It looks hard to understand.
So I suggest add `frozen columns` in LogColumns config.
So we can do something like this one:
```java
    for (int i = 0, n = columns.size(); i < n; ++i) {
        final String columnName = columns.get(i).getName();
        if (frozenColumns.contains(columnName)) {
          treeTable.addColumn(tableColumns.get(i));
        }
      }
```

Reply:
Interesting!  One thing to consider is that the "frozen" distinction is kind of odd (my bad).  I think a better design would not treat any columns so specially.  I think that would be clearer and more flexible down the line.

How about I open another PR in a bit where we can explore those ideas specifically?

Reply:
Yeap, Please!

Reply:
Perfect.  Busy few days but let's pick that up next week.  In the meantime, I did a few renames for you. 👍

Reply:
Thank you!